cdap-authorization-dataset-plugin/src/main/java/co/cask/cdap/security/authorization/DatasetBasedAuthorizer.java

```diff
@@ -67,11 +71,22 @@ public ACLDataset get() {
         return new ACLDataset(table);
       }
     };
+    Properties properties = context.getExtensionProperties();
+    if (properties.containsKey("superusers")) {
```

Comment:
Added a warning right now. We could perhaps add a check while implementing https://issues.cask.co/browse/CDAP-5327.